proposals/exp-show-target.md

```diff
@@ -0,0 +1,214 @@
+# Add targets/stages argument to dvc exp show
+
+- Enhancement Proposal PR: (leave this empty)
+- Contributors: dberenbaum, (add your Github handle)
+
+# Summary
+
+The `dvc exp show` command shows a comparison of the results of different
+experiments along with the parameters and metrics values for each experiment.
+The command's output currently includes:
+* All values that are defined in `params.yaml`.
+* All values in other referenced parameters files in any DVC stage.
+* All values in any referenced metrics files in any DVC stage.
+This proposal allows users to see only parameters and metrics relevant to
+specified pipeline stages.
```

Comment:
> specified pipeline stages

But `targets` would be an optional command argument right (dvc.yaml by default) ? Meaning that the user could still use `exp show` without specifying anything.

Reply:
Also this needs an extra line or will be interpreted as part of the last bullet:
```suggestion

This proposal allows users to see only parameters and metrics relevant to
specified pipeline stages.
```